posthog/warehouse/api/saved_query.py

```diff
@@ -100,8 +115,8 @@ def create(self, validated_data):
 
         with transaction.atomic():
             view.save()
-
             try:
+                sync_saved_query_workflow(view, create=True)
```

Comment:
I dont think we should auto sync on saved queries. We had considered this previously but opted against it so that users could still save queries as views just for the sake of having subqueries that are still run at query time. People could get confused why their view data is stale (even if only 24 hours by default).

Reply:
Yeah, I was modeling off the the migration for the sources which has a default interval in the db, but it does make sense to not default this to anything in the case that a user doesn't want to materialize.